pkg/clusterversion/cockroach_versions.go

```diff
@@ -257,6 +257,17 @@ const (
 	// the system tenant to ensure it is a superset of secondary tenants.
 	V24_1_AddSpanCounts
 
+	// V24_1 is CockroachDB v24.1. It's used for all v24.1.x patch releases.
+	V24_1
+
+	// V24_2Start_DoNotUseForVersionChecks demarcates the start of cluster versions
+	// stepped through during the process of upgrading from 24.1 to 24.2.
+	//
+	// NB: As of 24.2 we now support the expectation that test clusters should be
+	// safe to upgrade from one week to the next. This means that we should use
+	// new migrations over V24_2Start. Teams can still use IsActive(V24_2) placeholder.
+	V24_2Start_DoNotUseForVersionChecks
```

Comment:
@dt @RaduBerinde -
it's helpful to use `V24_2Start_DoNotUseForVersionChecks` instead of the usual `V24_2`, since we're still in the process of rolling out the new policy change?

(we can probably rename this back to plain ol' `V24_2Start` once the policy is more deeply ingrained into team's heads)

Reply:
What kind of checks would violate the policy we're considering? Currently I see it's used to check that the binary version is from that release series, which still seems valid.